osu.Game/Overlays/News/NewsCard.cs

```diff
@@ -0,0 +1,196 @@
+                                    Margin = new MarginPadding
+                                    {
+                                        Top = 10,
+                                        Right = 15
+                                    }
+                                }
+                            }
+                        },
+                        new Container
+                        {
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Padding = new MarginPadding
+                            {
+                                Horizontal = 15,
+                                Vertical = 10
+                            },
+                            Child = main = new TextFlowContainer
+                            {
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y
+                            }
+                        }
+                    }
+                },
+                new HoverClickSounds()
+            };
+
+            bg.OnLoadComplete += d => d.FadeIn(250, Easing.In);
+
+            main.AddParagraph(post.Title, t => t.Font = OsuFont.GetFont(size: 20, weight: FontWeight.SemiBold));
+            main.AddParagraph(post.Preview, t => t.Font = OsuFont.GetFont(size: 12)); // Should use sans-serif font
+            main.AddParagraph("by ", t => t.Font = OsuFont.GetFont(size: 12));
+            main.AddText(post.Author, t => t.Font = OsuFont.GetFont(size: 12, weight: FontWeight.SemiBold));
+        }
+
+        protected override bool OnHover(HoverEvent e)
+        {
+            background.FadeColour(colourProvider.Background3, 200, Easing.OutQuint);
+            return true;
+        }
+
+        protected override void OnHoverLost(HoverLostEvent e)
+        {
+            background.FadeColour(colourProvider.Background4, 200, Easing.OutQuint);
+            base.OnHoverLost(e);
+        }
+
+        [LongRunningLoad]
+        private class NewsBackground : Sprite
+        {
+            private readonly string sourceUrl;
+
+            public NewsBackground(string sourceUrl)
+            {
+                this.sourceUrl = sourceUrl;
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(LargeTextureStore store)
+            {
+                Texture = store.Get(createUrl(sourceUrl));
+            }
+
+            private string createUrl(string source)
+            {
+                if (string.IsNullOrEmpty(source))
+                    return "Headers/news";
+
+                if (source.StartsWith('/'))
+                    return "https://osu.ppy.sh" + source;
+
+                return source;
+            }
+        }
+
+        private class DateContainer : CircularContainer, IHasTooltip
+        {
+            public string TooltipText => date.ToString("d MMMM yyyy hh:mm:ss UTCz");
```

Comment:
This tooltip is not consistent with web design. This should be using whatever `DrawableDate` is, probably by hoisting out `DrawableDate.DateTooltip` of the inner class and making it public.